databases/week2/week2.sql

```diff
@@ -0,0 +1,119 @@
+
+
+------------TASK PART 1------------
+
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+SELECT * from task;
+insert into task (id, title, description, created, updated, due_date, status_id) value (38, 'Wash my clothes', 'Title says it all.', '2017-10-25 06:54:16', '2017-10-15 13:05:09', null, 2);
+--Change the title of a task
+UPDATE task  SET title= 'My clothes' WHERE task.title ='Wash my clothes';
+--Change a task due date// I changed all the nulls :( 
```

Comment:
Yes indeed, this query will not only change one specific task. The only way you can make sure that you only update a single row is if you use the primary key, since that is unique for all rows.